horace_core/sqw/@rundatah/rundatah.m

```diff
@@ -15,15 +15,17 @@
         % positions
         qpsecs_cache = []
     end
-    
+
     properties(Access=private)
         transform_sqw_f_=[];
     end
+
```

Comment:
comment for whole file, not specifically this line.

Missing the classVersion method.